examples/comid-3.diag

```diff
@@ -24,7 +24,7 @@
           / comid.mkey / 0: 700,
           / comid.mval / 1 : {
             / comid.digests / 2 : [[
-            / hash-alg-d / 6, / sha256_32 /
+            / hash-alg-d / 6, / sha-256_32 /
```

Comment:
the typo is still there...

Reply:
Yes, I had noticed it, will change it now!